packages/accounts-password/password_server.js

```diff
@@ -239,8 +239,18 @@ const checkForCaseInsensitiveDuplicates = (fieldName, displayName, fieldValue, o
   const skipCheck = Object.prototype.hasOwnProperty.call(Accounts._skipCaseInsensitiveChecksForTest, fieldValue);
 
   if (fieldValue && !skipCheck) {
+    let selector = {};
+    const { useLowerCaseEmails } = Accounts._options;
+    // if useLowerCaseEmails is true in Accounts.config and fieldName='emails.address'
+    // then don't run selectorForFastCaseInsensitiveLookup
+    // instead run a simple regex query for email.
+    if (useLowerCaseEmails && (fieldName === 'emails.address')) {
```

Comment:
The extra parenthesis is not needed here

Reply:
Hi I have made the necessary change